### ISSUE TITLE
US Interstate Numbering System

### ISSUE THREAD

**Issue by TheJoeFin** [fabiencelier/geoguessr-tips#7]
There is a helpful Wikipedia page on how interstate highways are generally numbered.

https://en.wikipedia.org/wiki/Interstate_Highway_System

Specifically the section Numbering system.

**Comment by fabiencelier** [comment]
I agree, this map is nice: https://en.wikipedia.org/wiki/Interstate_Highway_System#/media/File:FWHA_INS5901095-01.png

I will add it to the tips
